app-forensics/yara/yara-4.2.3.ebuild

```diff
@@ -26,7 +26,10 @@ DEPEND="
 RDEPEND="${DEPEND}"
 PDEPEND="python? ( =dev-python/yara-python-$(ver_cut 1)* )"
 
-PATCHES=( "${FILESDIR}/${PN}-$(ver_cut 1-2)-test.patch" )
+PATCHES=(
```

Comment:
No revbump required because it's purely a test issue _in the tests_. The tests need updating, but they expose no runtime problem, and the patch is only to the tests.